infra/resources/dev/italynorth/keyvault.tf

```
@@ -0,0 +1,18 @@
+resource "azurerm_resource_group" "sec_rg" {
+  name     = format("%s-sec-rg", local.project)
+  location = local.location
+
+  tags = local.tags
+}
+
+module "key_vault" {
+  source                     = "git::https://github.com/pagopa/terraform-azurerm-v3.git//key_vault?ref=v8.21.0"
+  name                       = format("%s-kv", local.project)
+  location                   = azurerm_resource_group.sec_rg.location
+  resource_group_name        = azurerm_resource_group.sec_rg.name
```

I would use the same resource group
```suggestion
  resource_group_name = azurerm_resource_group.rg_common.name
```